modules/ROOT/pages/vpc-tutorial.adoc

```diff
@@ -72,13 +72,15 @@ You can do this when you initially create the VPC, or you can configure the inte
 
 == Connect to your VPC
 
-xref:virtual-private-cloud.adoc#vpc-connectivity-methods[VPC connectivity methods] (IPSEC, AWS Peering, and so on) are not a self-serve function at this time. +
+xref:virtual-private-cloud.adoc#vpc-connectivity-methods[VPC connectivity methods] (VPC Peering and CloudHub Direct Connect) are not a self-serve function at this time. +
 To configure a connection between your CloudHub VPC and your infrastructure, download link:{attachmentsdir}/VPC-form-v9.3-template.xlsx[the VPC discovery form] (requires at least MS Excel 2007 with macros enabled), enter the necessary information required for your connectivity method and send it to MuleSoft customer support.
 
-The SLA for configuring VPC is 5 business days after the VPC discovery form is completed and returned, though it may be completed sooner.
+The SLA for configuring VPC is five business days after the VPC discovery form is completed and returned, though it may be completed sooner.
```

Comment:
What does "it" refer to? completing the configuration? We don't really say what we do after they submit the form. Do we twiddle things in the customer's org? Or enable them to do the actual configuring themselves?